USAGE.md

```diff
@@ -149,6 +150,14 @@ If you're looking to configure the amount of time between having to enter your K
 
 ![keychain-image](https://imgur.com/ARkr5Ba.png)
 
+### Pass
+
+**Note:** If using `pass` backend and you're experiencing the below error, you may need to `export GPG_TTY=$(tty)`
+```
+Error when retrieving credentials from custom-process: gpg: decryption failed: No secret key
+gpg: decryption failed: No secret key
+aws-vault: error: exec: Failed to get credentials for <your-profile>: exit status 2
+```
```

Comment:
> If using `pass` backend and you're experiencing the below error, you may need to

Can you explain the error, why it occurs, and why the suggested fix works?

Reply:
Honestly, I'd be guessing at best and it may be inaccurate. It seems that gpg-agent requires TTY info since the password prompt is an interactive curses like interface that fills up the whole screen after a refresh instead of just a `password:` prompt at a newline. 

When interacting with pass directly I didn't have the issue and I'm assuming its because my terminal preserves that tty info. But my suspicion is that the system / shell call from aws-vault to pass doesn't preserve or set that info, so the act of setting GPG_TTY in the env var allows it to fall back to that to get the tty device (and this info about the session and its cols/rows). 

Do you want me to put my guesstimate of why its needed in the markdown?

Reply:
I'd prefer not to put guesses into the USAGE, but if you explain what is going on then at least it can be corrected if not accurate

Reply:
> the system / shell call from aws-vault to pass doesn't preserve or set that info, so the act of setting GPG_TTY in the env var allows it to fall back to that to get the tty device

Wait.. this sounds like a bug in aws-vault then. Can you confirm this and fix in aws-vault directly?

Reply:
Sorry, I don't have any experience with Golang and don't have time to dive into it at this time. 

I understand if you don't want to accept the change based on the context. I just thought I'd contribute to try and save others the same lost time I had running into this issue.